src/omitBy.js

```diff
@@ -0,0 +1,23 @@
+import { complement, identity, pickBy, useWith } from 'ramda';
+
+/**
+ * Returns a partial copy of an object containing only the keys
+ * that don't satisfy the supplied predicate.
+ *
+ * @func omitBy
+ * @memberOf RA
+ * @category Object
+ * @sig ((v, k) -> Boolean) -> {k: v} -> {k: v}
+ * @param {!Function} pred A predicate to determine whether or not a key
```

Comment:
Don't wrap lines. We're using eslint comments to disable long line issues

+ drop the `.` character at the end of the sentence